cmd/provenanced/cmd/pre_upgrade_test.go

```diff
@@ -211,7 +210,6 @@ func makeDummyCmd(t *testing.T, home string) *cobra.Command {
 }
 
 func TestPreUpgradeCmd(t *testing.T) {
-	app.SetConfig(true, false)
```

Comment:
I removed this line because I couldn't see a reason why it would matter for this test and having it here was causing the `SimulateTestSuite` (in `cmd/provenanced/cmd/simulate_test.go`) to fail in setup.

I couldn't get it to happen locally, but it happened thrice in a row on the github action that runs our tests. Somehow, the `authority` address created in the bank module's `NewBaseKeeper` was getting one HRP, but by the time it was being checked in `NewBaseSendKeeper`, the other HRP was expected, so it was panicking. I suspect that the reason I couldn't get the failure locally was that the ordering of the tests in `github.com/provenance-io/provenance/cmd/provenanced/cmd` ends up being different in the github action vs locally.

The `AccAddress.String()` function uses some caching. At some point in these package tests, with the default HRP, something gets the gov module account address bech32 string. Later, this `app.SetConfig(true, false)` is called, changing the (global) HRP. But that `.String()` cache still has the old HRP for the given address bytes. So in the bank's `NewBaseKeeper`, the call to `authtypes.NewModuleAddress(govModuleName).String()` recognizes the bytes and returns the previously generated bech32 (with the now incorrect HRP). A few lines later, in `NewBaseSendKeeper`, it tries to decode that address, and finds the wrong HRP which makes it panic.

This was the only place where `SetConfig` was being called in this package's tests. Once I removed it, the tests started passing in the github action again.